airflow/api_connexion/schemas/sla_miss_schema.py

```diff
@@ -30,7 +30,8 @@ class Meta:
 
     task_id = auto_field(dump_only=True)
     dag_id = auto_field(dump_only=True)
-    execution_date = auto_field(dump_only=True)
```

Comment:
Should we still keep this for backward compatibility? This field is still present on the task instance schema, for example.

Reply:
Thought about it. I think it was removed in rtif so I followed the example. It's really only there as fk. Why would you want it here?

Reply:
Given SlaMiss is only ever included inside the response for a TI (and not currently requestible directly) we can loose this -- and e don't _need _task_id or dag_id either. But that's a bigger change.